lang/ilias_de.lang

```diff
@@ -16302,6 +16302,8 @@ tbl#:#tbl_template_deleted#:#Die gespeicherte Ansicht wurde gelöscht.
 tbl#:#tbl_templates#:#Ansicht
 tos#:#deleteDocument#:#Löschen
 tos#:#detachCriterionAssignment#:#Löschen
+tos#:#tos_accept_usr_agreement_anonymous#:#Nutzungsvereinbarung
+tos#:#tos_accept_usr_agreement_anonymous_intro#:#Wenn Sie zu ILIAS fortfahren, akzeptieren Sie die folgenden Nutzungsvereinbarungen.
```

Comment:
„fortfahren“ hört sich in diesem Zusammenhang etwas holperig an. Ich würde eher tendieren zu: „Wenn Sie zu ILIAS wechseln, akzeptieren Sie die folgenden Nutzungsvereinbarungen.“ Aber zum Ablauf eine Verständnisfrage. Akzeptiere ich hier etwas automatisch, dadurch, dass ich zu ILIAS wechsle? Oder muss ich vor dem Wechsel noch etwas akzeptieren? Im erstgenannten Fall würde ich präzisieren „... wechseln, akzeptieren Sie automatisch die folgenden …“. Im zweiten Fall wäre präziser: „Bevor Sie zu ILIAS wechseln, akzeptieren Sie …“.

Reply:
This is just a read only view and nothing is accepted neither automatically nor explicitly.
This is basically like a static info page where the ToS are shown.

Reply:
@matthiaskunkel just to clarify: IMO the affected page is neither the first nor the second case you suggested.
So should I use  `Bevor Sie zu ILIAS wechseln, akzeptieren Sie …` or do we need an entirely different sentence?

Reply:
Let's try „Bevor Sie zu ILIAS wechseln, akzeptieren Sie …“. I have to see the screen in real to know if it is what I am expecting... Changing the language text afterwards is not a big deal.

Reply:
That's true :) The text is adjusted.